src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -21,18 +21,18 @@ WMTS_Provider.prototype.customUrl = function customUrl(layer, url, tilematrix, r
 WMTS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer) {
     layer.fx = layer.fx || 0.0;
 
+    layer.format = layer.format || 'image/png';
     layer.options = layer.options || {};
 
     if (layer.protocol === 'wmts') {
         const options = layer.options;
         options.version = options.version || '1.0.0';
         options.tileMatrixSet = options.tileMatrixSet || 'WGS84';
-        options.mimetype = options.mimetype || 'image/png';
         options.style = options.style || 'normal';
         options.projection = options.projection || 'EPSG:3857';
         let newBaseUrl = `${layer.url}` +
             `?LAYER=${options.name}` +
-            `&FORMAT=${options.mimetype}` +
+            `&FORMAT=${layer.format}` +
```

Comment:
`options.mimetype` to `layer.format` to be in another PR, both must be supported with a deprecated message

Reply:
I can put that in another PR if you want, but I still don't see the need for a depreciation message. I did give my reasons, but you didn't give yours.

Reply:
I think it breaks the users cases

Reply:
Yes, but what about my suggestion in my previous comment? If it's clearly documented in the release notes, as it is a trivial change to do, user won't have any difficulty right? They would upgrade their itowns lib, and in the same commit rename this property. 

Anyway, I'll see if I can remove this from this PR.

Reply:
As it breaks the user cases, it's better, we have to support both and put a deprecated message

Reply:
Here it is: https://github.com/iTowns/itowns/pull/597